src/pages/messages/components/DeleteIconButton.tsx

```diff
@@ -0,0 +1,30 @@
+import DeleteIcon from '@mui/icons-material/Delete';
+import { IconButton, IconButtonProps, Tooltip } from '@mui/material';
+import { mainTheme } from '../../../themes/mainTheme';
+
+interface Props extends IconButtonProps {
+  tooltip?: string;
+}
+
+export const DeleteIconButton = ({ sx, tooltip, ...rest }: Props) => (
+  <Tooltip title={tooltip}>
+    <IconButton
+      sx={{
+        bgcolor: mainTheme.palette.primary.main,
```

Comment:
`sx` har noen shortcuts som gjør at man kan bruke verdier fra theme uten å måtte importere det

```suggestion
        bgcolor: 'primary.main',
```